README.md

```diff
@@ -1 +1,16 @@
-# To-do-list
\ No newline at end of file
+# To-do-list
+
+The user can add his daily to do here and when list item is completed he/she can remove it from the list or even edit it if he made a mistake. And also remove all of them in one try.
+
+![screenshot](version.png)
+
+## Built with:
+  * HTML
+  * Javascript
+  * CSS
+  * Webpack
+  * Fontawesome
+
+## Author
+ Follow me on: [LinkedIn](https://www.linkedin.com/in/firdavs-allamurotov-12b60a226/)  
+ Follow me on: [Github](https://github.com/fed1k)  
```

Comment:
- [ ] In your README, kindly add more detailed instructions on how users can set up and run your app locally. Note that users need to run `npm install` after cloning, they can run the other start or build commands.

- [ ] Add the **Prerequisites** section, as we have in the [Microverse template](https://github.com/microverseinc/readme-template) declaring the technologies users need to have before the app can be run.

 - [ ] For a much more elaborate README, I would recommend you add these other sections to your documentation.

  ![fdk-contr](https://user-images.githubusercontent.com/36542986/159973387-9f9689eb-e6b7-48cf-a55b-c95484a81ae9.jpg)